Packs/Cryptocurrency/Integrations/Cryptocurrency/Cryptocurrency.py

```diff
@@ -0,0 +1,92 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+import urllib3
+import traceback
+from typing import Dict, List
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+BITCOIN = 'bitcoin'
+INTEGRATION_NAME = 'Cryptocurrency'
+SCORE = {
+    4: 'Critical',
+    3: 'Bad',
+    2: 'Suspicious',
+    1: 'Good',
+    0.5: 'Informational',
+    0: 'Unknown'
+}
+
+
+def get_bitcoin_reputation(addresses) -> List[CommandResults]:
+    command_results: List[CommandResults] = []
+    score = 2
+    for address in addresses:
+        dbot_score = Common.DBotScore(
+            indicator=address,
+            indicator_type=DBotScoreType.CRYPTOCURRENCY,
+            integration_name=INTEGRATION_NAME,  # Vendor
+            score=score  # Suspicious
+        )
+        crypto_context = Common.Cryptocurrency(
+            address=address,
+            address_type=BITCOIN,
+            dbot_score=dbot_score
+        )
+        table_data = {
+            'Address': address,
+            'Cryptocurrency Address Type': BITCOIN,
+            'Reputation': SCORE[score]
+        }
+        table_name = f'{INTEGRATION_NAME} reputation for {address}'
+        hr = tableToMarkdown(table_name, table_data)
+
+        command_results.append(CommandResults(
+            outputs_prefix='Cryptocurrency',
+            readable_output=hr,
+            outputs_key_field='Address',
+            indicator=crypto_context
```

Comment:
here and everywhere
https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas
```suggestion
            indicator=crypto_context,
```